fortuna/src/marketplace/Marketplace.js

```diff
@@ -3,9 +3,58 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import '../Main.css';
+import ChassisView from './ChassisView.js';
+import CannonsViews from './CannonsView.js';
+import CasusBlocksView from './CasusBlocksView.js';
+import JammersView from './JammersView.js';
+import TreadsView from './TreadsView.js';
+import AccessoriesView from './AccessoriesView.js';
+import ScannersView from './ScannersView.js';
+
+
+type Props = {||}; 
+type State = {|
+	marketplaceView: React.Element<any>,
+|};
+
 
 // Marketplace component.
-class Marketplace extends React.Component<{||}> {
+class Marketplace extends React.Component<Props, State> {
+
+	constructor(){
+		super();
+		this.state={
+			marketplaceView: <h2>Select a type on the left</h2>
+		}
+	}
+	showMarketView(selectedTankPartType: string) {
```

Comment:
If there are a fixed number of strings that this is allowed to be, this needs to be a type (i. e. a type that isn't string). Why is this bad? What if someone called this method and passed in 'Cannons' for example? Flow would say that is just fine, but that clearly isn't acceptable input. The fact that it isn't should be indicated by its method signature.